server/auth.go

```diff
@@ -59,7 +59,9 @@ type ClientAuthentication interface {
 
 // NkeyUser is for multiple nkey based users
 type NkeyUser struct {
-	Nkey                   string              `json:"user"`
+	Nkey string `json:"user"`
+	// this is a copy of the issued at (iat) field in the jwt
```

Comment:
PLace comment at end to keep spacing same.

Reply:
Fixed